extensions/wikia/PhalanxII/Phalanx_setup.php

```diff
@@ -63,13 +63,15 @@
 			'EditContent'                     => 'editContent',
 			'CheckContent'                    => 'checkContent',
 			'APIEditBeforeSave'               => 'filterAPIEditBeforeSave',
+			'FileUploadSummaryCheck'          => 'checkContent',
 		),
 	'PhalanxTitleBlock' =>
 		array(
 			'EditFilter'                      => 'editFilter',
 			'CreateDefaultQuestionPageFilter' => 'checkTitle',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "checkTitle" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)